Model/OrderData.php

```diff
@@ -32,4 +32,17 @@ public function getOrderChunks($storeId)
         $totalOrders = $this->getOrderQuery($storeId)->getSize();
         return (int) ceil($totalOrders / $this->chunkItems);
     }
+    
+    public function getDeletedProducts($storeId) {
+        $query = "SELECT sales_order_item.item_id, sales_order_item.parent_item_id, sales_order_item.product_id,
```

Comment:
Would this query return unique deleted products?
Example:
Order X has "Deleted shirt size XS"
Order Y has "Deleted shirt size XS"

---

How many times would "Deleted shirt size XS" be sent to Metrilo?

Reply:
Just made couple of tests because i have missed that scenario (task is updated), but sadly this will sync the XS size 2 times, I am working on fix for this issue.